custom-vu/src/main/kotlin/jces1209/vu/page/DcIssueNavigator.kt

```diff
@@ -23,13 +25,31 @@ class DcIssueNavigator(
                 presenceOfElementLocated(By.id("issue-content"))
             ),
             presenceOfElementLocated(By.id("key-val")),
-            presenceOfElementLocated(By.className("issue-body-content"))
+            presenceOfElementLocated(By.className("issue-body-content")),
+            presenceOfElementLocated(By.xpath("//*[@class = 'show-filter-details']"))
         )
     )
         .timeout(Duration.ofSeconds(30))
         .serverErrors()
         .build()
 
+    private val filterSubscriptionFalliblePage = FalliblePage.Builder(
+        driver,
+        and(
+            presenceOfElementLocated(By.id("filter-subscription")),
+            presenceOfElementLocated(By.id("filter.subscription.prefix.daysOfMonth")),
+            presenceOfElementLocated(By.xpath("//*[contains(text(), 'Filter Subscription')]")),
+            presenceOfElementLocated(By.xpath("//*[contains(text(), 'Recipients')]")),
+            presenceOfElementLocated(By.xpath("//*[contains(text(), 'Schedule')]")),
+            presenceOfElementLocated(By.xpath("//*[contains(text(), 'Interval')]")),
+            presenceOfElementLocated(By.id("filter-subscription-submit")),
+            presenceOfElementLocated(By.id("filter-subscription-cancel"))
```

Comment:
Duplicate with Cloud

Reply:
But there is some difference.
presenceOfElementLocated(By.id("-dialog")),